Content.Client/GameObjects/Components/Mobs/Actions/ActionAssignments.cs

```diff
@@ -116,9 +116,12 @@ public ActionAssignments(byte numHotbars, byte numSlots)
             {
                 foreach (var (hotbar, slot) in slots)
                 {
-                    if (!assignment.TryGetItemActionWithItem(out var actionType, out _)) continue;
-                    AssignSlot(hotbar, slot,
-                        ActionAssignment.For(actionType));
+                    if (!assignment.TryGetItemActionWithItem(out var actionType, out _))
+                    {
+                        continue;
+                    }
+
+                    ClearSlot(hotbar, slot, false);
```

Comment:
it looks like removed items, which gave actions, from this list:
    public enum ItemActionType : byte
    {
        Error,
        ToggleInternals,
        ToggleLight,
        ToggleMagboots,
        DebugInstant,
        DebugToggle,
        DebugTargetPoint,
        DebugTargetPointRepeat,
        DebugTargetEntity,
        DebugTargetEntityRepeat
    }

were not properly cleared from action slots.

I'm not 100% sure that this is the proper solution, but I played around with a CE Hardsuit Helmet and an Oxygen tank, and they both seemed to get updated/cleared without any problems